go-tests/etcd-start.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Start etcd in a Docker container, and wait for it to be healthy.
+#
+
+SC_ETCD_VERSION=${SC_ETCD_VERSION:="3.5.17"}
```

Comment:
This default version is the version used in prod (https://github.com/Scalingo/appsdeck-kitchen/blob/ef7b503f9a9d31e9eb4529a8981d4d734b564230/environments/staging.rb#L103-L104)

Reply:
You are right, I'll update the procedure to include the fact that we need to update the version here:

https://www.notion.so/scalingooriginal/MOP-050-Upgrade-Etcd-d69392a9508d4b7e8bbb42db7b4da220?source=copy_link#febb1b5724554eb8b343f6fefbccba6a